hw/ip/pwrmgr/dv/pwrmgr_sim_cfg.hjson

```diff
@@ -59,6 +59,11 @@
       uvm_test_seq: pwrmgr_smoke_vseq
       run_opts: ["+test_timeout_ns=1000000"]
     }
+    {
+      name: pwrmgr_reset
+      uvm_test_seq: pwrmgr_reset_vseq
+      run_opts: ["+test_timeout_ns=1000000"]
```

Comment:
if most of tests are using this timeout value, we can put it in the pwrmgr_base_test.